fuse/node/mount_darwin.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"syscall"
 
+	"upspin.io/log"
```

Comment:
not right

Reply:
Oh yeah, that was goimports shooting me in the foot. Damn, I thought I caught all of these...